app/test/account/like_test.dart

```diff
@@ -33,12 +33,12 @@ void main() {
       final rs = await client.listPackageLikes();
       expect(rs.likedPackages.isEmpty, true);
 
-      final rs2 = await client.likePackage('hydrogen');
-      expect(rs2.package, 'hydrogen');
+      final rs2 = await client.likePackage(hydrogen.package.name);
+      expect(rs2.package, hydrogen.package.name);
       expect(rs2.liked, true);
 
-      final rs3 = await client.likePackage('helium');
-      expect(rs3.package, 'helium');
+      final rs3 = await client.likePackage(helium.package.name);
+      expect(rs3.package, helium.package.name);
       expect(rs3.liked, true);
 
       final rs4 = await client.listPackageLikes();
```

Comment:
nit. can we add `expect(rs.likedPackages.map((e) => e.package), contains(hydrogen.package.name));`...